peachjam/migrations/0097_auto_20230815_0818.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.19 on 2023-08-15 08:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("peachjam", "0096_auto_20230814_1557"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
This needs to be a rename operation, otherwise we'll lose the existing data.

Reply:
Django should give you that option when you create the migration -- it'll ask if you've renamed the field. Or just change this migration to be a rename.